optuna/samplers/_tpe/multi_objective_sampler.py

```diff
@@ -32,8 +34,129 @@ def _default_weights_above(x: int) -> np.ndarray:
     return np.ones(x)
 
 
+class _ParzenEstimatorParameters(
```

Comment:
This class is copied from `optuna.samplers._tpe.parzen_estimator._ParzenEstimatorParameters`.

Reply:
[just a question] Is it OK not to add some prefix for indicating that this parameter (and estimator) are for multi-objective TPE?

Reply:
And is it needed to create a file for the estimator like `_MultivariateParzenEstimator` or `_ParzenEstimatorParameters`?

Reply:
Thanks for the comment. These classes are only being added temporarily and will be integrated as the #2614 task progresses. Therefore, I'm not too concerned about naming conventions here.

Reply:
Mostly curious but could you possibly share more concretely how they will be merged together afterwards?

Reply:
Now, I plan to entirely remove `optuna/samplers/_tpe/multi_objectrive_sampler.py` in the 4th task in https://github.com/optuna/optuna/issues/2614. `_ParzenEstimatorParameters` and `_ParzenEstimator` are edited in #2618. I will remove the current ones in `multi_objective_sampler.py`.